native/dlaqr23.go

```diff
@@ -98,6 +101,9 @@ func (impl Implementation) Dlaqr2(wantt, wantz bool, n, ktop, kbot, nw int, h []
 			panic("lapack: invalid value of ihiz")
 		}
 	}
+	if recurse < 0 {
```

Comment:
> Does it need to be an int? Can't it be a bool?

I wasn't sure here. As far as LAPACK is concerned, it can be a bool because it (Dlaqr04) would be called only with 1 from Dhseqr. I thought that someone might want to play with higher levels of recursion although it has been reported that no significant gain comes from it, so this use case is rather unlikely. Let me know and I can change it.